components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/Device.kt

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.appservices.fxaclient
+
+data class Device(
+    val id: String,
+    val displayName: String,
+    val deviceType: Type,
+    val pushSubscription: PushSubscription?,
+    val pushEndpointExpired: Boolean,
+    val isCurrentDevice: Boolean,
+    val lastAccessTime: Long?,
+    val capabilities: List<Capability>
+) {
+    enum class Capability {
```

Comment:
I had a bit of a chuckle at this, because we used to call these "capabilities" on the server side until we needed more data and turned them into "commands".

If we end up removing support for non-megazord builds, does that clear a future path to separating the send-tab stuff out into its own distinct component, at which point we might not need "capability" flags like this inside the core FxA component?  (For the future obviously, this seems fine for now).

Reply:
> I had a bit of a chuckle at this, because we used to call these "capabilities" on the server side until we needed more data and turned them into "commands".

Yup my rationale is that the clients don't need the `Commands` "values" as the UI only care about "capabilities", but I might be wrong in the future :)

Yes it might make it easier, we should look into that later this year.